dronegen/windows.go

```diff
@@ -232,7 +233,28 @@ func buildWindowsTshStep(workspace string) step {
 			`Enable-Go -ToolchainDir "$Workspace` + toolchainDir + `"`,
 			`cd $TeleportSrc`,
 			`$Env:GCO_ENABLED=1`,
-			`go build -o build/tsh.exe ./tool/tsh`,
+			`go build -o build/tsh-unsigned.exe ./tool/tsh`,
+		},
+	}
+}
+
+func signTshStep(workspace string) step {
+	return step{
+		Name: "Sign tsh",
+		Environment: map[string]value{
+			"WORKSPACE_DIR":        {raw: workspace},
+			"WINDOWS_SIGNING_CERT": {fromSecret: "WINDOWS_SIGNING_CERT"},
+		},
+		Commands: []string{
+			`$ErrorActionPreference = 'Stop'`,
+			`$Workspace = "` + perBuildWorkspace + `"`,
+			`$TeleportSrc = "$Workspace` + teleportSrc + `"`,
+			`. "$TeleportSrc/build.assets/windows/build.ps1"`,
+			`cd $TeleportSrc`,
+			`([System.Convert]::FromBase64String($ENV:WINDOWS_SIGNING_CERT)) | Set-Content windows-signing-cert.pfx -Encoding Byte`,
+			`& 'C:\Program Files (x86)\Windows Kits\10\App Certification Kit\signtool.exe' sign /f windows-signing-cert.pfx /d Teleport /t http://timestamp.digicert.com /du https://goteleport.com /fd sha256 build\tsh-unsigned.exe`,
```

Comment:
This looks fine. Only concern is if the location of `signtool.exe` will change from version to version of VS code.

Reply:
i think it's not gonna change with vs code version but probably with system major update. I don't have though better idea how to search for it.